math/pronicnumber.go

```diff
@@ -0,0 +1,26 @@
+// pronicnumber.go
+// description: Returns true if the number is pronic, false otherwise
+// details: A number n is a pronic number if
+// it is equal to product of two consecutive numbers m and m+1.
+// wikipedia: https://en.wikipedia.org/wiki/Pronic_number
+// author: Akshay Dubey (https://github.com/itsAkshayDubey)
+// see pronicnumber_test.go
+
+package math
+
+// This function returns true if argument passed to the function is pronic, false otherwise.
+func PronicNumber(n int) (bool, error) {
+	switch {
+	case n < 0:
+		return false, ErrPosArgsOnly
```

Comment:
Returning an error doesn't make sense. The function's job is to check if the number is pronic so just return `false`. The `error` value is unnecessary overhead.

Reply:
Fixed in [this](https://github.com/TheAlgorithms/Go/pull/595/commits/fa17bc22ff61a99c28cb38d86f0a5a860ea45c9e) commit.